src/Cardano/Fee.hs

```diff
@@ -116,27 +106,6 @@ newtype DustThreshold = DustThreshold
     deriving stock (Eq, Generic, Ord)
     deriving Show via (Quiet DustThreshold)
 
-{-------------------------------------------------------------------------------
-                                Fee Calculation
--------------------------------------------------------------------------------}
-
--- | Compute fee for a given payload. Fee follows a simple linear
--- equation:
---
--- @
---     f = a + size * b
--- @
---
--- where @a@ & @b@ are values fixed by the protocol.
-computeFee
-    :: FeePolicy
-    -> Quantity "byte" Int
-    -> Fee
-computeFee policy (Quantity sz) =
-    Fee $ ceiling (a + b*fromIntegral sz)
-  where
-    LinearFee (Quantity a) (Quantity b) (Quantity _c) = policy
```

Comment:
I was surprised to find that this function isn't actually used by the fee balancing code.

Reply:
yeah, this is very wallet specific. we ingest fee policy and determine `Fee` . And `cardano-coin-selection` starts from this level ;-)